setup.py

```diff
@@ -1,17 +1,17 @@
 try:
-    from setuptools import setup,
+    from setuptools import setup
 except ImportError:
     from distutils.core import setup
 
 config = {
-    name='pynio',
-    version='0.1dev',
-    packages = ['pynio'],
-    install_requires = [
+    'name': 'pynio',
+    'version': '0.1dev',
```

Comment:
is it common to include "dev" in the version? I would prefer to leave it out, 0.1 is pretty explicit.